pkgs/development/python-modules/django-ninja/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, django
+, fetchFromGitHub
+, flit
+, psycopg2
+, pydantic
+, pytest-asyncio
+, pytest-django
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "django-ninja";
+  version = "0.22.2";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "vitalik";
+    repo = "django-ninja";
+    rev = "v${version}";
+    sha256 = "sha256-oeisurp9seSn3X/5jFF9DMm9nU6uDYIU1b6/J3o2be0=";
+  };
+
+  propagatedBuildInputs = [ django pydantic ];
+
+  nativeCheckInputs = [
+    flit
+    psycopg2
+    pytest-asyncio
+    pytest-django
+    pytestCheckHook
+  ];
+
+  meta = with lib; {
+    changelog = "https://github.com/vitalik/django-ninja/releases/tag/v${version}";
+    description = "Web framework for building APIs with Django and Python type hints.";
```

```suggestion
    description = "Web framework for building APIs with Django and Python type hints";
```
Please remove the trailing period.